tianshou/utils/net/common.py

```diff
@@ -44,17 +45,22 @@ class Recurrent(nn.Module):
     customize the network), please refer to :ref:`build_the_network`.
     """
 
-    def __init__(self, layer_num, state_shape, action_shape, device='cpu'):
+    def __init__(self, layer_num, state_shape, action_shape,
+                 device='cpu', size=128):
         super().__init__()
         self.state_shape = state_shape
         self.action_shape = action_shape
         self.device = device
-        self.nn = nn.LSTM(input_size=128, hidden_size=128,
+        self.nn = nn.LSTM(input_size=size, hidden_size=size,
                           num_layers=layer_num, batch_first=True)
-        self.fc1 = nn.Linear(np.prod(state_shape), 128)
-        self.fc2 = nn.Linear(128, np.prod(action_shape))
+        self.fc1 = nn.Linear(np.prod(state_shape), size)
+        self.fc2 = nn.Linear(size, np.prod(action_shape))
 
     def forward(self, s, state=None, info={}):
+        """In the evaluation mode, s should be with shape ``[bsz, dim]``; in
+        the training mode, s should be with shape ``[bsz, len, dim]``. See the
+        code and comment for more detail.
+        """
```

Comment:
``size`` is over-used. ``hidden_units`` may be better. The same in the other files.

Reply:
just a reference:

```python
from sklearn.neural_network import MLPClassifier
MLPClassifier?
Init signature:
MLPClassifier(
    hidden_layer_sizes=(100,),
...
```

So, maybe ``hidden_layer_size`` is better.